### PR TITLE
feat(challenge-redirect): Make '/challenge' and '/map' redirect to learn

### PR DESCRIPTION
**Pull request by Bouncey**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "lint-js": "eslint --ext=.js,.jsx gulpfile.js server/ common/ config/ client/",
     "lint-json": "npm run lint-server && npm run lint-challenges && npm run lint-resources && npm run lint-utils",
     "only-once": "npm run create-rev && echo '/****/' && echo 'Seeding Database' && echo '/****/' && node seed && echo '/****/' && echo 'Seeding Completed' && echo '/****/'",
-    "prelint-js": "npm run create-rev",
+    "prelint-js": "npm run create-rev && gulp generate-migration-map",
```
Comment (raisedadead) [inline package.json]: Can you make this more friendly, to our current contributors? This will most certainly crash setups for contributors?

**Review by raisedadead — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Bouncey) [reply package.json]: How so? `gulp` is no longer a global dependency

Comment (raisedadead) [reply package.json]: Well for one, it will not work when people are doing a fresh clone and `npm run only-once`

Comment (raisedadead) [reply package.json]: I think you would have to 

```
"only-once": "npm run create-rev && gulp generate-migration-map && ...
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Yup, this will crash if the json is not available before hand. As I tested on a different machine. `node seed` crashed

**Review by raisedadead — APPROVED** [review]

**Pull request merged**